hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -74,6 +74,7 @@ enum Type {
   DBUpdates = 53;
   FinalizeUpgrade = 54;
   FinalizeUpgradeProgress = 55;
+  Prepare = 56;
```

Comment:
Can we rename to PrepareUpgrade which can be more readable? In the design doc, it also called PrepareUpgrade request.

Reply:
Ignored this, see below comments about this.